### PR TITLE
After reaching 5 goals, the player wins, changes color to green, and the game resets

### PR DESCRIPTION
**Pull request by Aleksa-Jov**

### CODE REVIEW

File: source/dagger/gameplay/ping_pong/player_scores.cpp [hunk 1]
```diff
@@ -45,6 +46,39 @@ void PlayerScoresSystem::Run()
...
+
+    for (auto entity : view) 
+    {
+        auto &ball = view.get<PingPongBall>(entity);
+
+        if (m_goalsPlayerOne == 5 || m_goalsPlayerTwo == 5) 
+        {
+            if (ball.processed)
+            {
+                Engine::Registry().destroy(entity);
```
Comment (DPrivalov) [inline source/dagger/gameplay/ping_pong/player_scores.cpp]: it's a bad practice to destroy objects in a Run method. Check out suggestions about how to fix it in other pull request:
https://github.com/UbiBelETF/dagger/pull/40#discussion_r522999212

**Review by DPrivalov — CHANGES_REQUESTED** [review]
looks good! 
one change is required

**Review by tozica — APPROVED** [review]

File: source/dagger/gameplay/ping_pong/player_scores.cpp (hunk 1 above)
Comment (Aleksa-Jov) [reply source/dagger/gameplay/ping_pong/player_scores.cpp]: Ok, thanks for the advice!

**Review by DPrivalov — APPROVED** [review]
nice and clean

**Pull request merged**